src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelper.java

```diff
@@ -934,7 +934,10 @@ public static Address getRespondentAddressET3(RespondentSumType respondentSumTyp
 
         log.info("Get respondent address ET3");
 
-        return (respondentSumType.getResponseRespondentAddress() != null
+
+        return (!Strings.isNullOrEmpty(respondentSumType.getResponseReceived())
```

Comment:
This is getting a little complicated to understand. You could replace the first two lines with
`YES.equals(espondentSumType.getResponseReceived())`